src/events.jl

```diff
@@ -45,10 +45,10 @@ Displaz.event_loop(
 end
 ```
 """
-function event_loop(callback::Function, event_list...)
+function event_loop(plotobj::DisplazWindow, callback::Function, event_list...)
```

Comment:
Having the callback first means you can use the `do` syntax.

If `DisplazWindow` is a concrete type, perhaps it could be a keyword argument?